examples/files.json

```diff
@@ -127,6 +127,7 @@
 		"webgl_lod",
 		"webgl_marchingcubes",
 		"webgl_materials",
+		"webgl_materials_dissolve",
```

Comment:
I don't think we need a new example. Adding the method to `SceneUtils` is sufficient 👍 .